src/components/PosterCardDropTarget.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { DropTarget } from "react-dnd";
+import { successToast, errorToast } from "../utils/toast";
+
+/**
+ * This component is a drop zone for PosterCards
+ */
+class PosterCardDropTarget extends Component {
+  static propTypes = {
+    /** enables react-dnd */
+    connectDropTarget: PropTypes.func.isRequired,
+    /** is true if a PosterCard is hovering over */
+    isOver: PropTypes.bool.isRequired,
+    /** name of the drop target, used for the notification text */
+    targetName: PropTypes.string.isRequired, // eslint-disable-line
+    children: PropTypes.node.isRequired,
+  }
+
+  render() {
+    const { connectDropTarget, isOver, children } = this.props;
+    return connectDropTarget((
+      <div className={isOver ? "droptarget-is-over" : ""}>
+        {children}
+      </div>
+    ));
+  }
+}
+
+const listTarget = {
+  // is called when a PosterCard is dropped on this component
+  drop(props, monitor) {
+    const item = monitor.getItem();
+    const success = Math.random() > 0.3;
```

Comment:
We shouldn't merge test cases to master

Reply:
What should we have instead?

Reply:
It should always show that it's successful before we added real states where there could be errors.

Reply:
Okay, I added this mostly to show that `errorToast()` exists, but I'll fix this.